.github/workflows/browserstack-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: BrowserStack
+
+# BrowserStack tests are run against the deployed GitHub pages URL
+# The closest event we can use to trigger when these are updated is page_build.
+# on: [page_build]
+on: [push]
+
+jobs:
+  browserstack:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - config: 'Windows_10_latest'
```

Comment:
can we use version numbers here. Their implementation of these things isn't great.

Reply:
I'd quite like to keep up with evergreen browser versions if we can. This seems to be working here so are you happy if we trial it using latest and if we hit any issues we can lock the versions?